cognite_toolkit/cdf_tk/load/_data_loaders.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import io
+import re
+from pathlib import Path
+from typing import final
+
+import pandas as pd
+from cognite.client.data_classes import capabilities
+from cognite.client.data_classes.capabilities import Capability, FilesAcl, RawAcl, TimeSeriesAcl
+
+from cognite_toolkit.cdf_tk.utils import CDFToolConfig
+
+from ._base_loaders import DataLoader
+from ._resource_loaders import FileMetadataLoader, RawDatabaseLoader, RawTableLoader, TimeSeriesLoader
+from .data_classes import RawDatabaseTable
+
+
+@final
+class DatapointsLoader(DataLoader):
+    folder_name = "timeseries_datapoints"
+    filetypes = frozenset({"csv", "parquet"})
+    dependencies = frozenset({TimeSeriesLoader})
+
+    @classmethod
+    def get_required_capability(cls, ToolGlobals: CDFToolConfig) -> Capability:
+        scope: capabilities.AllScope | capabilities.DataSetScope = (
+            TimeSeriesAcl.Scope.DataSet([ToolGlobals.data_set_id])
+            if ToolGlobals.data_set_id
+            else TimeSeriesAcl.Scope.All()
+        )
+
+        return TimeSeriesAcl(
+            [TimeSeriesAcl.Action.Read, TimeSeriesAcl.Action.Write],
+            scope,
+        )
+
+    def upload(self, datafile: Path, dry_run: bool) -> tuple[str, int]:

```

Comment:
I don't think we should deviate from our terminology. We deploy to CDF projects, we don't upload. I think it should be consistent also because we don't to give the impression that we support full-scale uploads of data to CDF projects.

Reply:
Hmm, we have `.deploy_resources` on the base class that uses this `.upload` method. If we call this `.deploy` then the loading the object will have both a `.deploy_resources` and `.deploy` method making it unclear what the difference is. I think in this case, we are uploading a datafile is descriptive.